scripts/bundleSize/bundleSize.test.js

```diff
@@ -1,30 +1,57 @@
-import ora from 'ora';
-import chalk from 'chalk';
-import { readdirSync, statSync } from 'fs';
 import stripAnsi from 'strip-ansi';
+import { jest } from '@jest/globals';
+import pageTypeBundleExtractor from './__mocks__/pageTypeBundleExtractor.js';
 
-jest.mock('./pageTypeBundleExtractor');
-jest.mock('ora');
-jest.mock('chalk', () => ({
-  red: a => a,
-  green: a => a,
-  blue: a => a,
-  yellow: a => a,
-  cyan: { bold: b => b },
-  bold: a => a,
-}));
-jest.mock('../../cypress/support/config/services', () => ({
-  service1: {},
-  service2: {},
-}));
-jest.mock('fs');
-jest.mock('./bundleSizeConfig', () => ({
+jest.unstable_mockModule('./pageTypeBundleExtractor', () => (pageTypeBundleExtractor));
+
+jest.unstable_mockModule('./bundleSizeConfig', () => ({
   MIN_SIZE: 632,
   MAX_SIZE: 728,
 }));
 
+jest.unstable_mockModule(
+  './serviceList.js',
+  () => ({ default: ["service1", "service2"] }),
+);
+
+jest.unstable_mockModule('fs', () => ({
+  default: {
+    readdirSync: jest.fn(),
+    statSync: jest.fn(),
+  },
+}));
+
+const start = jest.fn();
+const succeed = jest.fn();
+const fail = jest.fn();
+
+jest.unstable_mockModule('ora', () => {
+  const mock = jest.fn();
+  mock.mockReturnValue({
+    start,
+    succeed,
+    fail,
+  });
+  return {
+    default: mock,
+  };
+});
+
+jest.unstable_mockModule('chalk', () => ({
+  default: {
+    red: a => a,
+    green: a => a,
+    blue: a => a,
+    yellow: a => a,
+    cyan: { bold: b => b },
+    bold: a => a,
+  },
+}));
+
 const setUpFSMocks = (service1FileSize, service2FileSize) => {
-  beforeEach(() => {
+  beforeEach(async () => {
+    const { default: fs } = await import('fs');
```

Comment:
`jest.mock` works by hoisting all require calls to the top of the file and jest.mock would then override the imported object with your configured mock.

From the jest docs:
> Since ESM evaluates static import statements before looking at the code, the hoisting of jest.mock calls that happens in CJS won't work for ESM. 
https://jestjs.io/docs/ecmascript-modules#module-mocking-in-esm

We therefore need to do a 'dynamic import' allowing jest to intercept the module resolution process and inject the mock we configure. Here we do this to utilise the mock configured for the `fs` module here: https://github.com/bbc/simorgh/pull/10937/files#diff-186540c5991ff0121d0c796909193883e3289c4b84c8d327af446173a6423ecfR17